util/syspolicy/syspolicy_test.go

```diff
@@ -43,6 +43,20 @@ func (th *testHandler) ReadUInt64(key string) (uint64, error) {
 	return th.u64, th.err
 }
 
+func (th *testHandler) ReadBoolean(key string) (bool, error) {
+	if key != string(th.key) {
+		th.t.Errorf("ReadBool(%q) want %q", key, th.key)
+	}
+	var value bool
+	switch th.u64 {
+	case 0:
+		value = false
+	case 1:
+		value = true
+	}
+	return value, th.err
```

Comment:
```suggestion
	return value != 0, th.err
```

... or add a bool field